pkg/cmd/server/start/start_master.go

```diff
@@ -558,7 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder(binderClient)
 		kc.RunPersistentVolumeProvisioner(provisionerClient)
-		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient)
+		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, bootstrappolicy.InfraPersistentVolumeRecyclerControllerServiceAccountName, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
```

Comment:
above, we build the recyclerClient to use the recycler controller service account already... you don't need to also give the pod the powers of that service account